openmetadata-service/src/main/resources/json/data/testConnections/database/dynamodb.json

```diff
@@ -1,15 +1,15 @@
 {
-    "name": "DynamoDB",
-    "fullyQualifiedName": "DynamoDB",
-    "displayName": "DynamoDB Test Connection",
-    "description": "This Test Connection validates the access against the database and basic metadata extraction of schemas and tables.",
-    "steps": [
-      {
-        "name": "GetTables",
-        "description": "List all the tables available to the user.",
-        "errorMessage": "Failed to fetch tables, please validate the credentials & validate if user has access to fetch tables",
-        "shortCircuit": true,
-        "mandatory": true
-      }
-    ]
-  }
\ No newline at end of file
+  "name": "DynamoDB",
+  "fullyQualifiedName": "DynamoDB",
+  "displayName": "DynamoDB Test Connection",
+  "description": "This Test Connection validates the access against the database and basic metadata extraction of schemas and tables.",
+  "steps": [
+    {
+      "name": "ListTables",
```

Comment:
Not 100% sure, do we need to migrate this or not?

Reply:
we will need it